rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -361,13 +497,19 @@ s32 cellCameraOpenEx(s32 dev_num, vm::ptr<CellCameraInfoEx> info)
 	g_camera->is_open = true;
 	g_camera->info = *info;
 
+	cellCamera.success("cellCameraOpenEx() returned CELL_OK");
```

Comment:
this also applies to log messages, also why not "cellCameraOpenEx() succeeded"

Reply:
that's garbage that i forgot to clean